style.css

```diff
@@ -0,0 +1,89 @@
+* {
```

Comment:
- [ ] Your design does not match the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). The proposed branch, _**feature-styles**_ which you want to merge does not match with the screenshot of the design you've provided in your ReadMe. Perhaps, this issue is due to the wrong CSS configuration or a wrong branch submitted. Kindly ensure to improve upon this aspect.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/133285746-b9188a09-ecba-4977-9eab-8c587fe5d29e.png)


**The original wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/133285637-3516641b-02b4-44a9-9e5b-c294c7544975.png)

Reply:
Kindly take a look at it again @rloterh

![1](https://user-images.githubusercontent.com/58692590/133287875-29682ff8-0bb2-4a1d-b839-ad4b3a9eb893.png)